Example/ExampleApp/Questionnaire/ORKOrderedTaskView.swift

```diff
@@ -24,13 +25,44 @@ struct ORKOrderedTaskView: UIViewControllerRepresentable {
     init(
```

Comment:
If would be great if we can add documentation for the `tintColor` and `outputDirectory` parameters to fully document the initializer.

Reply:
Good catch! Addressed by [0adf95f](https://github.com/StanfordBDHG/ResearchKitOnFHIR/pull/59/commits/0adf95ff479a250443d7d294353aa94839993a9c)